GNUmakefile

```diff
@@ -41,13 +41,9 @@ cov:
 	gocov test ${GOFILES} | gocov-html > /tmp/coverage.html
 	open /tmp/coverage.html
 
-test:
-	@./scripts/verify_no_uuid.sh
-	@env \
-		GOTAGS="${GOTAGS}" \
-		GOFILES="${GOFILES}" \
-		TESTARGS="${TESTARGS}" \
-		sh -c "'$(CURDIR)/scripts/test.sh'"
+test: dev
+	go test -tags "$(GOTAGS)" -i -run '^$$' ./...
```

Comment:
There's a `GOFILES` thing we could use here as well as `TEST` for the above (to get `./...` but let it be hooked into). Or just gut those if you want to keep it simple. I personally run the tests outside the Makefile with `go test` when I'm not running them all, since we don't need any non-go-standard stuff to run these.